src/Traits/SoftDeletes.js

```diff
@@ -2,10 +2,13 @@
 
 const Database = use('Database')
 const { retrieveTableDetails } = require('../utils')
-
+const _ = require('lodash')
 class SoftDeletes {
-  register (Model) {
-    const deletedAtColumn = 'deleted_at'
+  register (Model, customOptions) {
+    const defaultOptions = {deletedAtColumn: 'deleted_at'}
+    const options = _.extend({}, defaultOptions, customOptions)
```

Comment:
I don't like the idea of importing `lodash` only to extend a single object. You should use object spread operator for that:

```js
const options = {
  deletedAtColumn: 'deleted_at',
  ...customOptions,
}
```